doc/src/site/sphinx/deployment/backends.rst

```diff
@@ -101,40 +101,13 @@ lead (as in the previous mode) to h2o cluster being killed as well.
 
 There are two deployment strategies of the external cluster: manual and automatic. In manual mode, we need to start
 the H2O cluster, and in automatic mode, the cluster is started for us automatically based on our configuration.
-In both modes, we can not use the regular H2O driver jar as the deployment artifact for the external H2O cluster.
+In hadoop environments, the creation of the cluster is performed by a simple process called H2O driver.
+When the cluster is fully formed, the H2O driver terminates. In both modes, we have to store a path of H2O driver jar
+to the environment variable ``H2O_DRIVER_JAR``.
 
-Obtaining Extended H2O Jar
-^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-The extended H2O jar can be downloaded using our helper script available in the official Sparkling Water distribution which
-can be downloaded from `http://www.h2o.ai <http://www.h2o.ai/>`_.
-After you download and unpack the Sparkling Water distribution package, you can use the ``./bin/get-extendend-h2o.sh``
-script to download the extended H2O jar. This script expects a single argument that specifies the Hadoop version
-for which you obtained the jar.
-
-The following code downloads H2O extended JAR for the cdh5.8:
-
- .. code:: bash
-
-    ./bin/get-extended-h2o.sh cdh5.8
-
-If you don't want to run on Hadoop and instead want to run H2O in standalone mode, you can get the corresponding
-extended H2O standalone jar as:
-
- .. code:: bash
-
-    ./bin/get-extended-h2o.sh standalone
-
-If you want to see a list of supported Hadoop versions, just run the shell script without any arguments as:
-
- .. code:: bash
-
-    ./bin/get-extended-h2o.sh
-
-The script downloads the jar to the current directory and prints the absolute path to the downloaded jar.
+.. code:: bash
 
-**Note**: If you want to get an extended H2O jar for Sparkling Water and H2O versions that have not yet been
-released, you need to extend the JAR manually. This is explained in the following tutorial: :ref:`extend_jar_manually`.
+    H2O_DRIVER_JAR=/path/to/h2o-driver.jar
```

Comment:
Should this section still mention how to obtain the JAR -> `./bin/get-h2o-driver.sh`?

Reply:
Good idea, will change it!